src/components/project/home-block.vue

```diff
@@ -262,10 +262,15 @@ export default {
     "encrypted": "Encrypted",
     "encryptionTip": "This Project uses managed encryption.",
     // This clickable text is shown below a table of forms where only a few out of the total number ("count") of forms is shown.
-    "showMore": "Show {count} total",
+    "showMore": "Show {count} total Forms",
```

Comment:
I know it will always be multiple forms, but I think this message (and the others) still need to be pluralized because some languages pluralize differently based on the number. There will end up being an English singular case here which is never shown.